aliases/cowsay.yaml

```diff
@@ -1,9 +1,21 @@
 # this is command alias for `actions/cowsay.yaml` action
-# allows to run ChatOps command: `cowsay "Holy ChatOps Cow!"`
-# eq: cowsay "Holy ChatOps Cow!"
+# allows to run ChatOps command: `cowsay Holy ChatOps Cow!`
+# eq: `/usr/games/cowsay Holy ChatOps Cow!`
 ---
 name: chatops.cowsay
 action_ref: st2-chatops-aliases.cowsay
 description: Draws a cow that says what you want
 formats:
-  - "cowsay {{message}}"
+  - display: "cowsay <message>"
+    representation:
+      - "cowsay {{ message }}"
+ack:
+  enabled: false
+result:
+  format: |
+    {% if execution.status == 'succeeded' %}Here is your cow: ```{{ execution.result.stdout }}``` {~}{% else %}
```

Comment:
@emedvedev Here, for example If I include new line after the `{~}`, eg:

```yaml
  format: |
    {% if execution.status == 'succeeded' %}
    Here is your cow: ```{{ execution.result.stdout }}``` {~}
    {% else %}
    Sorry, no cows this time {~}
    Exit Code: `{{ execution.result.return_code }}`
    Stderr: ```{{ execution.result.stderr }}```
    Hint: Make sure `cowsay` utility is installed.
    {% endif %}
```

then I see "no real mention" and empty attachment after the main message:
![screenshot from 2016-06-26 16-13-32](https://cloud.githubusercontent.com/assets/1533818/16362511/3af098dc-3bb9-11e6-9d7e-fbadb5e5982b.png)